src/components/DataBrowserHeaderBar/DataBrowserHeaderBar.react.js

```diff
@@ -76,7 +76,9 @@ export default class DataBrowserHeaderBar extends React.Component {
     if (onAddColumn) {
       const finalStyle = {};
       if (headers.length % 2) {
-        finalStyle.background = 'rgba(224,224,234,0.10)';
+        finalStyle.background = '#726F85';
+      }else {
+        finalStyle.background = '#66637A';
```

Comment:
I can see that color in `global.scss` - `$pushDetailsContent`.

Reply:
I also noticed that on line 44, the same color '#66637A' is being used directly:
wrapStyle.background = '#66637A';
This code was present in the original file and wasn't part of my recent changes

![image](https://github.com/user-attachments/assets/80b426f9-a7b4-4b5b-93c3-51b70c56dedc)
soo should i also change it ?

Reply:
Go ahead if you want 👍